lib/server.js

```diff
@@ -189,3 +203,21 @@ module.exports = {
   }
 };
 
+function cleanUpAndExit() {
+  if (electronPids) {
```

Comment:
Nope. If I remove the other lines of the codes, the electron window won't shut down gracefully. I'm on `Node v5.11.1`. What's your node version?